suite-common/test-utils/src/configureMockStore.ts

```diff
@@ -30,8 +31,21 @@ export function configureMockStore<S = any, A extends Action = AnyAction>({
 } = {}) {
     let actions: A[] = [];
 
-    const actionLoggerMiddleware = createMiddleware<A>((action, { next }) => {
-        actions.push(action);
+    const actionLoggerMiddleware = createMiddleware((action, { next }) => {
+        if (action?.meta?.requestId) {
+            // requestId is generated random string, and it will break fixtures because they are static, so we remove it
+            if (action?.meta?.arg === undefined) {
+                // only requestId and requestStatus are left, remove meta completely
+                actions.push(D.deleteKey(action, 'meta') as any);
```

Comment:
What's the problem with this?

```suggestion
                actions.push(D.deleteKey(action, 'meta') as A);
```

Reply:
This is not working, generic A doesn't have meta property and it will throw

Reply:
It could be probably typed little bit better but for these test utils it doesn't matter that much, in worse case it will break some unit tests